CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add `the-things-stack` device template converter, enabled by default. Effectively, this allows importing end devices from the Console.
+- GCS endpoint to download global_conf.json.
```

Comment:
You also need to mention the added config option that operators need to set on their deployments when upgrading.